spaces/domains.json

```
@@ -610,4 +610,5 @@
   "vote.orion.security": "orionsecurity.eth",
   "governance.xdc.network": "xinfinxdcnetwork.eth",
   "vote.komorebinosekai.com": "komorebi-no-sekai.eth"
+  "vote.rug.farm": "saynotorugs.eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "vote.komorebinosekai.com": "komorebi-no-sekai.eth",
  "vote.rug.farm": "saynotorugs.eth"
```